disnake/threads.py

```diff
@@ -460,7 +460,24 @@ def permissions_for(
         parent = self.parent
         if parent is None:
             raise ClientException("Parent channel not found")
-        return parent.permissions_for(obj, ignore_timeout=ignore_timeout)
+        # n.b. GuildChannel is used here so implicit overrides are not applied based on send_messages
+        base = GuildChannel.permissions_for(parent, obj, ignore_timeout=ignore_timeout)
```

Comment:
with these changes, this part of the documentation is no longer entirely accurate:
https://github.com/DisnakeDev/disnake/blob/eff3638c0c409e27ab2dd9a3b447ae482ca0b139/disnake/threads.py#L425-L428
Maybe clarify that it handles `send_messages_in_threads` specially?

Reply:
[`b87c854` (#1047)](https://github.com/DisnakeDev/disnake/pull/1047/commits/b87c8542e3a22cbdf7880873f524df39d9ef1858)